<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Atividade 02</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>-1</y>
      <width>491</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;ÁREA DE DESENHO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxColor">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>19</y>
      <width>351</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>351</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="toolTip">
           <string>Selecione uma figura geométrica e edite suas propriedades correspondentes.</string>
          </property>
          <item>
           <property name="text">
            <string>Clear</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Círculo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Retângulo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Reta</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Elipse</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Retângulo com bordas arredondadas</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="widgetSwitcher">
          <property name="currentIndex">
           <number>4</number>
          </property>
          <widget class="QWidget" name="pageCirculo">
           <widget class="QSpinBox" name="circuloRaio">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>250</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_6">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>20</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Raio</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="circuloCentroY">
            <property name="geometry">
             <rect>
              <x>130</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>80</y>
              <width>61</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>Centro</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="circuloCentroX">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>250</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>100</y>
              <width>21</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>x</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>210</x>
              <y>100</y>
              <width>61</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>y</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="pageRetangulo">
           <widget class="QSpinBox" name="retanguloL1">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
           <widget class="QSpinBox" name="retanguloL2">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>40</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
           <widget class="QSpinBox" name="retanguloL3">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>375</number>
            </property>
           </widget>
           <widget class="QSpinBox" name="retanguloL4">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>375</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>50</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P1</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_3">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>50</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P2</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>110</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P3</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_5">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>110</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P4</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="pageReta">
           <widget class="QSpinBox" name="retaL1">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_14">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>40</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Y1</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="retaL2">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>30</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_15">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>40</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>X1</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_16">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>110</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Y2</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_17">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>110</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>X2</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="retaL4">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>450</number>
            </property>
           </widget>
           <widget class="QSpinBox" name="retaL3">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>450</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_18">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>150</y>
              <width>201</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>*Você não verá a cor branca.</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="pageElipse">
           <widget class="QSpinBox" name="elipseL1">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>40</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>35</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_10">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>50</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P1</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="elipseL2">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>40</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>175</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>50</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P2</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="elipseL3">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>400</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>110</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P4</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="elipseL4">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>100</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_13">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>110</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P3</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="pageRoundedRect">
           <widget class="QSpinBox" name="roundedL4">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>80</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>375</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_19">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>90</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P3</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_20">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>30</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P2</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_21">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>30</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P1</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="roundedL2">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>20</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
           <widget class="QSpinBox" name="roundedL1">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>20</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_22">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>90</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>P4</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="roundedL3">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>80</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>480</number>
            </property>
            <property name="value">
             <number>375</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_23">
            <property name="geometry">
             <rect>
              <x>270</x>
              <y>150</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>X2</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="roundedX2">
            <property name="geometry">
             <rect>
              <x>180</x>
              <y>140</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
           <widget class="QLabel" name="label_24">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>150</y>
              <width>49</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>X1</string>
            </property>
           </widget>
           <widget class="QSpinBox" name="roundedX1">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>140</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
